winlogtimeline/ui/help_window.py

```diff
@@ -0,0 +1,69 @@
+from tkinter import *
+from tkinter.ttk import *
+
+
+class HelpWindow(Toplevel):
+    def __init__(self, parent, type):
+        super().__init__(parent)
+
+        # Class variables
+        self.project_path = None
+
+        # Window parameters
+        self.resizable(width=False, height=False)
+
+        # Create and place the widgets
+        self._init_widgets()
+        self._place_widgets()
+
+        if type == "about":
+            self.title('About')
+            self.text.config(text="The PyEventLogViewer project was developed over the course\n"
+                                  "of six months from late 2017 to early 2018 as part of a sponsored\n"
+                                  "course at Cal Poly, San Luis Obispo. There were four main\n"
+                                  "contributors to this project: Carter Van Deuren, Zach Monroe,\n"
+                                  "Sarah Stephens, and Shane Kent. At the time of developing this\n"
+                                  "project, all the contributors were studying toward receiving\n"
+                                  "their BS in Computer Engineering from Cal Poly, San Luis Obispo.")
```

Comment:
I'd like to see version info in this window, but I don't expect you to include that at the moment since we aren't following a version scheme.